packages/office-ui-fabric-react/src/utilities/grid/Grid.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { BaseComponent, getId } from '../../Utilities';
+import { IGridProps } from './Grid.Props';
+
+export class Grid extends BaseComponent<IGridProps, {}> {
+
+  // Used to track the index of item we are on while building grid
+  private _gridIndex;
+
+  private _id;
+
+  private _items: any[][];

```

Comment:
Yep, good catch. That was from when I was making the transition in the code. I've updated this